test/Proxy.test.js

```diff
@@ -69,9 +69,9 @@ function startProxyServers() {
 
 describe('Proxy', () => {
   // issue: https://github.com/webpack/webpack-dev-server/pull/1633#issuecomment-463938675
-  if (shouldSkipTestSuite()) {
-    return;
-  }
+  // if (shouldSkipTestSuite()) {
+  //  return;
+  // }
```

Comment:
Let's see on issue status with updated `http-proxy-middleware`

Reply:
it seems good with Node11 🎉 https://travis-ci.org/webpack/webpack-dev-server/jobs/495415842

Reply:
@evilebottnawi no :smile: https://github.com/webpack/webpack-dev-server/pull/1672